students/ZidanLuo/session09/donor_models.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+import sys
+import os
+
+class Donor_Collection():
+    def __init__(self):
+        self.donor_dict = {"Fred Jones": Donor("Fred Jones", 500)}
```

Comment:
you don't want to hard-code any data in the __init__ -- if you want test data (which you do :-) ) -- add it later with a separate function that you can call in the tests, or for examples.